app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
Agreed that it's irrelevant in production (and documented as such), and conceptually it shouldn't be needed.  My personal criteria for how to approach a need is weighted more toward how to isolate the concern, avoid repetition, and simplest code, and I find it pragmatic at times to compromise some ideological principles.  

If you feel strongly that the create logic doesn't belong here, then I'll replace with a fatal exception and add the necessary code at the test spec level to guarantee the root node exists.